### DISCUSSION TITLE
The systems contains different atoms

### DISCUSSION THREAD

**Discussion by chenyouheng** [deepmodeling/deepmd-kit#236] [Q&A] [answered]
If the one system is composed of pure Co while another system is composed with Co3W, how can I train these two systems in one model? And the length of rel_a is 1 or 2?

**Comment by jameswind** [comment, marked as answer]
The length of rel_a is 2. The types of atoms are specified in type.raw. If
you use 0 for Co and 1 for W, then in one system all the atoms in type.raw
are labelled with type 0. In the other system both 0 and 1 appear.

**Comment by chenyouheng** [comment]
But my training goes wrong when the length of rel_a is 2 and the system contains only one W. However, If I change the type.raw so that it contains Co and W both, the training goes well.

**Comment by njzjz** [comment]
You can set `type_map` in the parameters to `["Co", "W"]`.

https://github.com/deepmodeling/deepmd-kit/blob/59d780c77aaf0c0692d1bb56a630c50bc987ea13/examples/water/train/water_se_a.json#L3-L4
